allocations/templates/allocations/index.html

```diff
@@ -159,11 +159,23 @@ <h4><span class="fa fa-spinner fa-spin"></span><span ng-show="isLoading('approve
                                         <div ng-show="!isEmpty(getMessages('approveAllocation', allocation))">
                                             <alert ng-repeat="msg in getMessages('approveAllocation', allocation)" type="[[msg.type]]">[[msg.body]]</alert>
                                         </div>
-                                         <div ng-show="!isEmpty(getMessages('rejectAllocation', allocation))">
+                                        <div ng-show="!isEmpty(getMessages('rejectAllocation', allocation))">
                                             <alert ng-repeat="msg in getMessages('rejectAllocation', allocation)" type="[[msg.type]]">[[msg.body]]</alert>
                                         </div>
                                     </td>
                                 </tr>
+                                <tr>
+                                    <td colspan="3">
+                                        <div class="progress">
+                                            <div class="progress-bar" role="progressbar" 
+                                            aria-valuenow=[[100 * allocation.computeAllocated / allocation.computeRequested]]
```

Comment:
I think what we want here is `computeUsed / computeAllocated`.

`computeRequested`: how many SUs the user asked for in their request (defaults to 20k, I don't think users can even change this at request time currently.)
`computeAllocated`: how many SUs were ultimately allocated to them. Normally this == `computeRequested`
`computeUsed`: how many SUs have been used so far. This includes pending leases to avoid you from reserving tons of stuff and not having the credit to back it up :)
`computeBalance`: how many SUs are left. I have no idea why we store this separately, it is the value of allocated - used.